packages/pwa-kit-react-sdk/package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "prepare": "npm run build",
     "build": "cross-env NODE_ENV=production internal-lib-build build",
-    "dev:watch": "watch 'npm run dev:build' src/",
+    "build:watch": "watch 'npm run build' src/",
```

Comment:
A small fix to fix this npm script (the issue also exists in `develop` branch btw). It wasn't working because it's pointing to `dev:build` script that no longer exists. And I also renamed the script.

I didn't make a separate PR for it.. hopefully that's all right. But I'm gonna make a PR to fix the issue in `develop` branch.